File: C_PROGRAMMING/ASSIGNMENTS/Assignment_3/SumNum.c
#include<stdio.h>

void main(){

 //print sum numbers from 1 to 5

   int start ,end ,sum = 0;
   printf("enter the start and end of the number ");
   scanf("%d%d",&start ,&end );
   while(start <=end)
   {
   	    sum = sum + start;
	    start++;
    }
        printf("\n%d",sum);
}

File: Java_Programs/Assignment_3/SumNum.java
package Assignment_3;

public class SumNum {

	public static void main(String[] args) {
		int start=1 ,end=10 ,sum = 0;
		   System.out.printf("enter the start and end of the number ");
		   while(start <=end)
		   {
		   	    sum = sum + start;
			    start++;
		    }
		   System.out.printf("\n%d",sum);
	}

}
